### ISSUE TITLE
Feature request: Vite.js version

### ISSUE THREAD

**Issue by casually-creative** [themeselection/sneat-bootstrap-html-admin-template-free#23]
### What problem does this feature solve?

Being able to use it in modern projects.

People are moving away from webpack and are more and more using vite.js. I wanted to use this template but it is too opinionated and reliant on webpack and gulp. Vite is where the world is moving towards. Would love to know if this is on the roadmap.

Bonus would be a vite.js + react + typescript (bootstrap) version.

**Comment by ajaypatelaj** [comment]
@casually-creative 

Thanks for purchasing our theme and getting in touch with us.
Our current structure is using Gulp + WebPack, yes the world is moving to vite.js and we would love to adopt the new technology.

Our next major release with structural changes will consider that.

We appreciate your feedback for vite.js + react + typescript (bootstrap), currently, we're providing the [Sneat NextJS](https://themeselection.com/item/sneat-mui-react-nextjs-admin-template/) version with TS  support. We don't have any plan for ReactStrap (React + Bootstrap).

Thanks & regards,
Ajay Patel

**Comment by casually-creative** [comment]
@ajaypatelaj 

Thank you for your reply.

Happy to know that you are considering moving towards vite. Looking forward to hearing more news about this in the future.

About your Sneat React + NextJS version, I was wondering if we can use the React components without the server side next.js?

For example can we take the components and use them client side with our own custom vite implementation?

Thx for your help!

**Comment by vrushank** [comment]
Hello @casually-creative,

Thanks for your reply. I sincerely appreciate your prompt response. After careful consideration, we have decided to align our development approach with React's recommendation to utilize Next.js. Additionally, we conducted a survey among our users, and Next.js emerged as the clear favorite with an overwhelming preference rate of over 90%. Consequently, we have decided to discontinue our efforts to make our template compatible with Vite.

Currently, we are dedicated to enhancing the compatibility of our Next.js app directory routing. This entails a comprehensive rewrite of our codebase. While it may be possible to adapt it for use with Vite by implementing the necessary modifications, regrettably, we won't be able to provide any support in this regard.

Thank you once again for your understanding.

Regards.
